src/patch-events.js

```diff
@@ -275,6 +275,10 @@ export function addEventListener(type, fn, optionsOrCapture) {
           return;
         }
       }
+      if(fn.handleEvent) {
```

Comment:
So you want to change the condition so that it checks that `typeof fn !== 'function'`?